docs/gcloud-auth.rst

```diff
@@ -58,15 +58,66 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful
```

Comment:
@jgeewax The semantic line-breaks (http://rhodesmill.org/brandon/2012/one-sentence-per-line/) got clobbered by a new version of the commit.

----

My last remark was that it was difficult / nebulous for me to do, it made it hard to keep up with PR comments and that I wasn't sure the process was worth it given the lines are all already under 80.